cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+COPY ./examples/hotrod/hotrod-demo-linux /bin/
```

Comment:
I don't think we want to do that. This image is meant to run jaeger-backend only. The demo can be run from source, especially since for the purpose of actual demos it requires changing the source.